java_gen/templates/of_class.java

```diff
@@ -425,7 +425,7 @@ public boolean equals(Object obj) {
         return true;
     }
 
-    //:: if filter(lambda m: m.name == 'xid', msg.data_members):
+    //:: if list(filter(lambda m: m.name == 'xid', msg.data_members)):
```

Comment:
I _think_ a possibly nicer way to express this in python could be:
```suggestion
    //:: if any(lambda m: m.name == 'xid', msg.data_members):
```
But I would want to check with the python experts bench @ronaldchl @JanStrauss - thoughts?

Reply:
assuming you mean "any(map(lambda ..., msg.data_members))", that seems to work

Reply:
Argh, what I meant a generator expression
```
any(m.name == “xid” for m in msg.data_members)
```

Reply:
ok, that also seems to work